release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -364,6 +364,20 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-networking"]
 === Networking
 
+[id="ocp-4-14-networking-ovn-kubernetes-ipsec-support-for-external-traffic"]
+==== OVN-Kubernetes network plugin support for IPsec encryption of external traffic general availability (GA)
+
+{product-title} now supports encryption of external traffic, also known as _north-south traffic_. IPsec already supports encryption of network traffic between pods, known as _east-west traffic_. You can use both features in conjunction to provide full in-transit encryption for {product-title} clusters.
```

Comment:
Would recommend using "together" rather than "in conjunction" for simplified English.